src/pages/registerPage/Register.tsx

```diff
@@ -1,8 +1,512 @@
+                  }}
+                  required
+                />
+              </InputComponent>
+              <InputComponent>
+                <Label htmlFor="name">
+                  Họ và tên <span> (*)</span>
+                </Label>
+                <TextField
+                  {...register('name')}
+                  helperText={errors.name?.message && errors.name.message}
+                  type="text"
+                  id="name"
+                  placeholder="Họ và tên"
+                  sx={{
+                    width: '100%'
+                  }}
+                  required
+                />
+              </InputComponent>
+              <InputComponent>
+                <Label htmlFor="birthday">
+                  Ngày sinh <span> (*)</span>
+                </Label>
+                <Controller
+                  control={control}
+                  {...register('birthday')}
+                  name="birthday"
+                  render={({ field: { value, ...fieldProps } }) => (
+                    <LocalizationProvider dateAdapter={AdapterDayjs}>
+                      <DateBirthday>
+                        <DatePicker
+                          {...fieldProps}
+                          label="Ngày sinh"
+                          value={value}
+                          className="inputBirthday"
+                          renderInput={(params) => <TextField {...params} />}
+                        />
+                      </DateBirthday>
+                    </LocalizationProvider>
+                  )}
+                />
+              </InputComponent>
+              <InputComponent>
+                <Label htmlFor="gender">
+                  Giới tính <span> (*)</span>
+                </Label>
+                <TextField
+                  {...register('gender')}
+                  helperText={errors.gender?.message && errors.gender.message}
+                  type="text"
+                  id="gender"
+                  placeholder="Giới tính"
+                  sx={{
+                    width: '100%'
+                  }}
+                  required
+                />
+              </InputComponent>
+              <InputComponent>
+                <Label htmlFor="province">
+                  Tỉnh/Thành phố <span> (*)</span>
+                </Label>
+                <Controller
+                  {...register('provinceId')}
+                  control={control}
+                  render={({ field, fieldState }) => (
+                    <FormControl fullWidth>
+                      <InputLabel>Tỉnh/Thành phố</InputLabel>
+                      <Select
+                        id="province"
+                        {...field}
+                        onChange={(event) => {
+                          field.onChange(event.target.value);
+                        }}>
+                        {provinces.map((province, index) => (
+                          <MenuItem key={index} value={province.id}>
```

Comment:
```suggestion
                          <MenuItem key={province.id} value={province.id}>
```
Chú ý: key nếu có id thì dùng id, ko có mới dùng index